src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -558,11 +555,11 @@ def post_process_panoptic_segmentation(
                             segments.append(
                                 {
                                     "id": current_segment_id,
-                                    "category_id": pred_class,
-                                    "is_thing": not is_stuff,
+                                    "label_id": pred_class,
```

Comment:
This change will break pipelines actually.

Tagging @sgugger for visibility of this BC.

The pipeline addition is so new that I am OK aligning here for sure.

Reply:
Let me know, I changed them to reflect the new naming

Reply:
The pipeline for Maskformer was not included in the release I believe, so ok to break IMO!